src/ImageSharp/Image/Image.Create.cs

```diff
@@ -7,56 +7,46 @@ namespace ImageSharp
 {
     using ImageSharp.PixelFormats;
 
-    /// <summary>
-    /// Represents an image. Each pixel is a made up four 8-bit components red, green, blue, and alpha
-    /// packed into a single unsigned integer value.
-    /// </summary>
+    /// <content>
+    /// Adds static methods allowing the creation of new images from given dimensions.
+    /// </content>
     public sealed partial class Image
     {
         /// <summary>
-        /// Create a new instance of the <see cref="Image{TPixel}"/> class
-        /// with the height and the width of the image.
+        /// Create a new instance of the <see cref="Image{TPixel}"/> class with the given height and the width.
         /// </summary>
-        /// <typeparam name="TPixel">The pixel format.</typeparam>
         /// <param name="width">The width of the image in pixels.</param>
         /// <param name="height">The height of the image in pixels.</param>
-        /// <param name="metadata">The images matadata to preload.</param>
         /// <param name="configuration">
         /// The configuration providing initialization code which allows extending the library.
         /// </param>
+        /// <typeparam name="TPixel">The pixel format.</typeparam>
         /// <returns>
-        /// A new <see cref="Image{TPixel}"/> unless <typeparamref name="TPixel"/> is <see cref="Rgba32"/> in which case it returns <see cref="Image" />
+        /// A new <see cref="Image{TPixel}"/>.
         /// </returns>
-        internal static Image<TPixel> Create<TPixel>(int width, int height, ImageMetaData metadata, Configuration configuration)
+        internal static Image<TPixel> Create<TPixel>(int width, int height, Configuration configuration)
```

Comment:
we can get rid of these now, now we don't have to do the `if (typeof(TPixel) == typeof(Rgba32))` thing, and just call the constructor directly from the `IImageDecoder`s